luatest/server.lua

```diff
@@ -93,6 +97,10 @@ end
 --   `net.box` connection to the new server.
 -- @tab[opt] object.box_cfg Extra options for `box.cfg()` and the value of the
 --   `TARANTOOL_BOX_CFG` env variable which is passed into the server process.
+-- @string[opt] object.config_file Declarative YAML configuration for server
+--   instance. Used to deduce advertise URI to connect net.box to the instance.
```

Comment:
The original `luatest.server` wrapper has a comment regarding the special `''` value.

Reply:
Fixed.